core/src/apps/bitcoin/sign_tx/matchcheck.py

```diff
@@ -107,6 +107,18 @@ def attribute_from_tx(self, txio: TxInput | TxOutput) -> Any:
         return multisig.multisig_fingerprint(txio.multisig)
 
 
+class PubkeysOrderChecker(MatchChecker):
+    def attribute_from_tx(self, txio: TxInput | TxOutput) -> Any:
+        if not txio.multisig:
+            return None
+        return txio.multisig is not None and txio.multisig.pubkeys_order
```

Comment:
Left hand side is always true, is it intentional?

Reply:
Not intentional, fixed in https://github.com/trezor/trezor-firmware/pull/4351/commits/7ed1c424dfdd7a4619c811763a28e2d78f0743dd.